### DISCUSSION TITLE
Were the rules for moving from mysql to mysqli removed?

### DISCUSSION THREAD

**Discussion by CViniciusSDias** [rectorphp/rector#8415] [Q&A] [answered]
Some old projects, before PHP 7.0, use the ext-mysql to connect to databases. There were rules to upgrade from ext-mysql to ext-mysqli but I can't find them anywhere in the repo anymore. Were they removed? Are there any alternatives?

**Comment by samsonasik** [comment, marked as answer]
Yes, that's removed, see PR:

- https://github.com/rectorphp/rector-src/pull/4448

for the reasoning.
